Libraries/Cordio/ble-host/sources/stack/att/atts_main.c

```diff
@@ -262,8 +262,10 @@ static void attsConnCback(attCcb_t *pCcb, dmEvt_t *pDmEvt)
       /* clear prepare write queue */
       attsClearPrepWrites(pAttsCb);
 
-      /* stop service discovery idle timer, if running */
-      if (DmConnCheckIdle(pCcb->connId) & DM_IDLE_ATTS_DISC)
+	  /* always stop service discovery idle timer(s) */
+	  /* if idle timer timed has already timed out then DM_IDLE_ATTS_DISC won't be set.
+	   * extra calls to WsfTimerStop() ok because it will just skip inactive timers */
+      //if (DmConnCheckIdle(pAttsCb->connId) & DM_IDLE_ATTS_DISC)
       {
         WsfTimerStop(&pAttsCb->idleTimer);
       }
```

Comment:
When I have the bandwidth a deeper look on the robustness of unscheduled disconnects is warranted on our end. For the time being this fix is good and verified. Just go ahead and remove the extranious curly brackets and were good to go. 
@khpeterson

Reply:
Thanks @EdwinFairchild, done.